library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +71,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    Set<DayViewDecorator> dayViewDecorators;
```

Comment:
I mean adding twice the same decorator by calling ```addDecorator(decorator)``` when you are adding/removing decorators dynamically. Otherwise there should be some way to check whether a decorator is added or not (```hasDecorator(decorator)``` or something similar). WDYT?